controllers/test/types.go

```diff
@@ -4,13 +4,11 @@ import (
 	"context"
 	"fmt"
 
-	"k8s.io/apimachinery/pkg/runtime"
-
-	v1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
 	"github.com/golang/protobuf/ptypes/empty"
 	"google.golang.org/grpc"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
please group go imports

Reply:
What principle or standard should I follow to group go imports? I tried `make groupimports` and it says it's ok.

Reply:
It looks well. Where is the problem? @fewdan